src/main/java/racing/view/ResultView.java

```diff
@@ -1,25 +1,23 @@
-package view;
+package racing.view;
 
-import racing.CarPositions;
-import racing.RacingResultDto;
-
-import java.util.List;
+import racing.model.CarsAtTurn;
+import racing.model.RacingResultDto;
 
 public class ResultView {
 
     public static void printResult(RacingResultDto racingResultDto) {
         System.out.println("실행 결과");
-        for (CarPositions carPositions : racingResultDto.getPlayHistory().getPositions()) {
-            printTurnResult(racingResultDto.getCarNames(), carPositions);
+        for (CarsAtTurn carsAtTurn : racingResultDto.getPlayHistory().getPositions()) {
```

Comment:
```suggestion
        for (CarsAtTurn carsAtTurn : racingResultDto.carAtTurns()) {
```

[디미터 법칙](https://www.google.com/search?q=%EB%94%94%EB%AF%B8%ED%84%B0%EB%B2%95%EC%B9%99)에 대해 고민해보아요~

Reply:
@wooobo 
`PlayHistory`가 iterator를 사용하게 수정하여 호출을 간소화했습니다.